packages/react-native-editor/ios/DocumentsMediaSource.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+import Gutenberg
+import MobileCoreServices
+
+class DocumentsMediaSource: NSObject {
+    private var mediaPickerCallback: MediaPickerDidPickMediaCallback?
+    private unowned var gutenberg: Gutenberg
+    private unowned var uploadCoordinator: MediaUploadCoordinator
+
+    init(gutenberg: Gutenberg, coordinator: MediaUploadCoordinator) {
+        self.gutenberg = gutenberg
+        self.uploadCoordinator = coordinator
+    }
+
+    func presentPicker(origin: UIViewController, filters: [Gutenberg.MediaType], multipleSelection: Bool, callback: @escaping MediaPickerDidPickMediaCallback) {
+        let uttypeFilters = filters.compactMap { $0.typeIdentifier }
+        mediaPickerCallback = callback
+        let docPicker = UIDocumentPickerViewController(documentTypes: uttypeFilters, in: .import)
```

Comment:
This initializer looks like it was recently deprecated in iOS 14: https://developer.apple.com/documentation/uikit/uidocumentpickerviewcontroller

Reply:
Nice catch!
We will need to update WPiOS too at some point 🤔

Reply:
Is this something best left out of here? Maybe we can create an issue for it and come back later, wdyt?